Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -303,6 +303,17 @@ class CredentialsManagerSpec: QuickSpec {
                 }
             }
             
+            context("custom keychain", closure: {
```

Comment:
For consistency there is no need to add the closure param name.
You can use `context("custom keychain") { .... }`

The storage property can then be set within the scope of this block so it will be accessible for `beforeEach` and the `it` tests, which also removes the need for the nil coalescing operator when assigning storage.